tamr_unify_client/client.py

```diff
@@ -97,7 +102,8 @@ def request(self, method, endpoint, **kwargs):
         :return: HTTP response
         :rtype: :class:`requests.Response`
         """
-        url = urljoin(self.origin + "/" + self.base_path, endpoint)
+
```

Comment:
usually there is no blank line after docstrings... please remove this blank line